gramps/plugins/lib/libgedcom.py

```diff
@@ -772,6 +772,7 @@
 for __val, __key in PERSONALCONSTANTEVENTS.items():
     if __key != "":
         GED_TO_GRAMPS_EVENT[__key] = __val
+        GED_TO_GRAMPS_EVENT["Stillbirth"] = EventType.STILLBIRTH
```

Comment:
I think you can just accept this suggestion and everything will be as you and @Nick-Hall suggested:

```suggestion
GED_TO_GRAMPS_EVENT["Stillbirth"] = EventType.STILLBIRTH
```